### PR TITLE
Fix rubocop issues

### PR DESCRIPTION
**Pull request by mtyaka**
This fixes issues detected by the most recent (0.52.1) rubocop release.

### CODE REVIEW

**Comment by pointlessone** [comment]
I'm pretty sure not all of this apply to all Ruby versions we support. Try setting target Ruby version for Rubocop to the oldest supported one. I believe, it's 2.2 at the moment.

**Comment by mtyaka** [comment]
Thanks @pointlessone! If I set target ruby version to 2.2, the `FrozenStringLiteralComment` no longer applies, but the others still do. I'll remove the `FrozenStringLiteralComment` change and rebase.

**Comment by mtyaka** [comment]
Rebased from a060f5f03f90e878497ec56d6124a5fc694e6e1f.

**Comment by pointlessone** [comment]
Could you please make the last commit a separate PR?

**Comment by mtyaka** [comment]
Build failed because TravisCI was still configured to run ruby 2.1. I removed 2.1 from travis config and added 2.5 instead.

**Comment by mtyaka** [comment]
@pointlessone 
> Could you please make the last commit a separate PR?

Sure, will do.

**Comment by mtyaka** [comment]
Rebased from 3a9eda873b5696f0a041f6cc0af7dfbc9d17a4ec.

File: ttfunk.gemspec [hunk 1]
```diff
@@ -5,7 +5,6 @@ Gem::Specification.new do |spec|
   spec.platform = Gem::Platform::RUBY
   spec.summary = 'TrueType Font Metrics Parser'
   spec.description = 'Font Metrics Parser for the Prawn PDF generator'
-  spec.licenses = %w[Nonstandard GPL-2.0 GPL-3.0]
```
Comment (pointlessone) [inline ttfunk.gemspec]: Could you please keep this and remove the other one?

**Comment by mtyaka** [comment]
Rebased from 0703d4ee66f05683da68ab92f8967336a6e48404.

File: ttfunk.gemspec (hunk 1 above)
Comment (mtyaka) [reply ttfunk.gemspec]: Sure, done.

**Comment by pointlessone** [comment]
Thank you for your contribution.

**Pull request merged**